### DISCUSSION TITLE
Can´t re-verify my student account

### DISCUSSION THREAD

**Discussion by Brianva7** [community/community#65718] [GitHub Education]
### Hi everyone,

When i try to re-verify my account whit the email im using for school i recieve this message: 

"has already been used to obtain an academic discount on GitHub account(s)"

![request](https://github.com/community/community/assets/112740750/d3db4649-ef4a-4199-9c9e-fd6518d6b121)

can anybody help me? 

### A suggestion...(optional)

see more students showing their projects!

**Comment by arafajs** [comment]
GitHub does not allow you to use the same email address to request an academic discount for multiple accounts.

**Reply by Brianva7** [reply]
i only have this account. is this an error?

**Reply by ianchb** [reply]
As you can see in the screenshot, GitHub requires students at your educational institution to use an email issued by that institution. You can consider changing/adding the email issued by the school before applying, or try adding some description after the existing school name (such as its city location) when filling in the school, so that GitHub should think that what you filled in is " Another” school. Of course, your info still needs to be truthful.

Hope that helps.
